src/test/java/core/basesyntax/RegistrationServiceImplTests.java

```diff
@@ -0,0 +1,142 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTests {
+    private static final String LOGIN_DEFAULT = "yurii.storozhuk@gmail.com";
+    private static final String PASSWORD_DEFAULT = "12345678";
+    private static final int PASSWORD_MIN_LENGTH = 6;
+    private static final int AGE_MIN = 18;
+    private static final int AGE_MAX = 122;
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private User user;
+
+    @BeforeEach
+    void setUpDefaultUser() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        user = new User(LOGIN_DEFAULT, PASSWORD_DEFAULT, AGE_MIN);
+    }
+
+    @Test
+    void register_userPasswordEqualsNull_Ok() {
+        user.setPassword(null);
+        Assertions.assertThrows(RegistrationServiceException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void register_userPasswordMinLength_Ok() {
+        User actual = storageDao.add(user);
+        Assertions.assertEquals(user, actual);
+    }
```

Comment:
this test is absolutely identical to register_userAgeMin_Ok() combine them to 1 overall success case